exampleSite/content/jobs/ranged/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+menu:
+  main:
+    name: Ranged
+    parent: jobs
+role: dps
```

Comment:
I actually think this is `role: ranged` – at least, that's what I've got working with production content. Double-check with @Roxas92 and @ScrubbyXIV though

Reply:
Ok let me take it away and check